File: BH Tic Tac Toe NetBeans Project/src/org/bh/game/ttt/local/UnsupportedLanguageException.java
package org.bh.game.ttt.local;

import java.util.Locale;

/**
 * UnsupportedLanguageException, made for BH Tic Tac Toe, is copyright Blue Husky Programming ©2014 GPLv3<HR/>
 * 
 * @author Kyli of Blue Husky Programming
 * @version 1.0.0
 * @since 2014-09-22
 */
class UnsupportedLanguageException extends Exception
{

	public UnsupportedLanguageException(Locale locale, Throwable cause)
	{
		super("Language not supported: " + locale, cause);
	}
	
}

File: BH Tic Tac Toe IntelliJ Project/src/org/bh/game/ttt/local/UnsupportedLanguageException.kt
package org.bh.game.ttt.local;

import java.util.*

/**
 * UnsupportedLanguageException, made for BH Tic Tac Toe, is copyright Blue Husky Software ©2014 GPLv3<HR/>
 * 
 * @author Kyli of Blue Husky Software
 * @version 2.0.0
 * @since 2014-09-22
 */
internal class UnsupportedLanguageException(locale: Locale, cause: Throwable) : Exception("Language not supported: " + locale, cause)
